src/components/Selector.js

```diff
@@ -0,0 +1,27 @@
+import styles from "./selector.module.scss";
+
+export default function Selector({
+  labelText,
+  navigate,
+  value,
+  arrayOfValues,
```

Comment:
You don't need to specify the type of the value in the name. So please rename it to just 'values'